uber/templates/preregistration/check_if_preregistered.html

```diff
@@ -2,26 +2,30 @@
 {% block title %}Check Registration for {{ c.EVENT_NAME }}{% endblock %}
 {% block content %}
 
-<div align="center"><h3>Check Preregistration</h3></div>
-<div>If for any reason you are unsure that you're preregistered for {{ c.EVENT_NAME_AND_YEAR }}, please enter your email
-    address below.
+<div class="masthead"></div>
```

Comment:
The ``masthead`` and ``panel panel-default`` divs were missing from this page, which is why it was all gray.  Probably due to the bootstrap refactor.